fineract-provider/src/main/java/org/apache/fineract/infrastructure/core/service/GmailBackedPlatformEmailService.java

```diff
@@ -61,6 +64,18 @@ public void sendDefinedEmail(EmailDetail emailDetails) {
         final String authuser = smtpCredentialsData.getUsername();
         final String authpwd = smtpCredentialsData.getPassword();
 
+        LOG.info("Use name: ", smtpCredentialsData.getUsername());
+        LOG.info("Use Password: ", smtpCredentialsData.getPassword());
```

Comment:
Why are you logging password and some critical information ? You can do this in testing on your Dev environment and then remove these lines as critical information is being exposed here.

Reply:
I haven't configured smtp on my pc. @francisguchie can use these logs to test the scenario in his own machine. This needs to be a draft PR. I'll change this into a draft one :)

Reply:
You can run Fineract on your local machine. Run the front end and test your changes with the default gmail service without setting up your own smtp on your PC. Am I missing something here? advice

Reply:
Let me try this sir :)

Reply:
@awasum @francisguchie  I tested this by creating a new user. All configuration values were null. Those values are getting from mysql database.

Reply:
@awasum Would you mind explaining the way those system configurations(smtp) were done?